### DISCUSSION TITLE
Welcome to PercepNet Discussions!

### DISCUSSION THREAD

**Discussion by jzi040941** [jzi040941/PercepNet#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by domcross** [comment]
Do you have any idea for how long (steps/epochs) the model should be trained, the paper isn't very specific here?

**Reply by jzi040941** [reply]
for my 500h dataset from DNS-Challenge model converged at 80k steps with 32 batch size

**Reply by jzi040941** [reply]
but when it comes to hearing the denoised output between 30k steps and 100k steps don't have significant difference
